sdcm/cluster.py

```diff
@@ -181,6 +179,27 @@ def set_test_id(cls, new_test_id):
         if new_test_id:
             cls._test_id = new_test_id
 
+    @classmethod
+    def logdir(cls):
```

Comment:
What happen if we will "reuse" already created cluster?

Reply:
I've test reuse, and it works found a new issue #977, but will take care of later